### PR TITLE
fix: nft background css

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes #mf-67

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -53,5 +53,23 @@
         "editor.defaultFormatter": "esbenp.prettier-vscode",
         "editor.wordWrap": "on",
         "editor.quickSuggestions": true
-    }
+    },
+    "i18n-ally.localesPaths": [
```
Comment (guanbinrui) [inline .vscode/settings.json]: ?

File: packages/mask/src/social-network-adaptor/twitter.com/injection/NFT/NFTAvatarClip.tsx [hunk 2]
```diff
@@ -49,16 +49,22 @@ function NFTAvatarClipInTwitter() {
...
-        setTimeout(() => {
+        const timer = setTimeout(() => {
             linkDom.current = searchTwitterAvatarNFTLinkSelector().evaluate()
             if (linkDom.current?.firstElementChild && linkDom.current?.childNodes.length === 4) {
                 borderElement.current = linkDom.current.firstElementChild
                 // remove useless border
                 linkDom.current.removeChild(linkDom.current?.firstElementChild)
             }
+            const link = linkDom.current as HTMLElement
+            if (link.style) {
```
Comment (hanszhoou) [inline packages/mask/src/social-network-adaptor/twitter.com/injection/NFT/NFTAvatarClip.tsx]: `HTMLElement` has defined property `style`, so remove this if condition?

**Comment by guanbinrui** [comment]
@lelenei Please ref an issue no.

**Review by guanbinrui — APPROVED** [review]

**Comment by guanbinrui** [comment]
@DimensionDev/qa

**Comment by AinuX** [comment]
Checked

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**